packages/core/src/payment/strategies/braintree/braintree-payment-options.ts

```diff
@@ -173,6 +173,7 @@ export type BraintreeFormFieldStyles = Partial<
 
 export interface BraintreeFormFieldKeyboardEventData {
     fieldType: string;
+    errors?: any;
```

Comment:
I suppose we should be able to type this out better as well.

Reply:
Good call 👍